src/Rectangle.js

```diff
@@ -129,6 +129,35 @@ class Rectangle {
     get height () {
         return Math.abs(this.top - this.bottom);
     }
+
+    /**
+     * Will pass a vector for each integer point within the bounds to a test function.
+     * If any testFunction returns truthy, the iteration will stop and return true.
+     * Otherwise, this will return false.
+     *
+     * @param {function} testFunction(point) should return true if iteration should stop.
+     * @param {?twgl.v3|array<number>} vec Storage for a vector to use for the iteration.
+     * @return {boolean} true if any testFunction returned true.
+     */
+    somePoint (testFunction, vec = [0, 0, 0]) {
```

Comment:
`twgl.v3.create()`

Reply:
I think more canonical names (in math libraries) for this method might be `overlapsPoint` or `intersectsPoint`?

Reply:
it's an iterator?  it's basically `points().some()` so I called it `.somePoint()`

I also didn't `twgl.v3.create` here because rectangle didn't already import twgl - and I don't think it cares what you use as an iterator.

Reply:
Hmm. Rather I suppose the iterator would be a `iteratePoints` member. The rectangle isn't defined as a collection of points but a bounding shape on a plane which can have points inside. Those can be infinite though so you need to define parameters on the set of points you want and `Symbol.iterator` will not have that option.